client/app/(auth)/sign-in/page.tsx

```diff
@@ -0,0 +1,124 @@
+'use client'
+
+import { Button } from '@/components/ui/button'
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from '@/components/ui/card'
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form'
+import { Input } from '@/components/ui/input'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { authAtom } from '@/lib/auth'
+import { useSetAtom } from 'jotai/react'
+import Link from 'next/link'
+import { useRouter } from 'next/navigation'
+import { useForm } from 'react-hook-form'
+import { toast } from 'sonner'
+import { loginFormSchema, type LoginFormSchema } from '@/types/login-register'
+import { fetchUser } from '@/lib/request'
+
+export default function SignInPage() {
+  const router = useRouter()
+  const setIsAuth = useSetAtom(authAtom)
+
+  // Define form
+  const form = useForm<LoginFormSchema>({
+    resolver: zodResolver(loginFormSchema),
+    defaultValues: {
+      username: 'test',
```

Comment:
Why is it prefilled?
![image](https://github.com/tientrinh21/uni-pov-exchange/assets/69582478/6e181c10-b838-4f5d-be36-72dc6c8eac73)

Reply:
![image](https://github.com/tientrinh21/uni-pov-exchange/assets/69582478/0848b8d1-972f-4112-9d70-9214e62a9a8d)

Reply:
Oh I prefilled it for testing, shall I get rid of it?